src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -69,6 +69,7 @@ api_platform:
         Sylius\Bundle\ApiBundle\Exception\ProductVariantCannotBeRemoved: 422
         Sylius\Bundle\ApiBundle\Exception\PromotionCouponCannotBeRemoved: 422
         Sylius\Bundle\ApiBundle\Exception\ProvinceCannotBeRemoved: 422
+        Sylius\Component\Core\Exception\ResourceDeleteException: 422
```

Comment:
This one could be added in alphabetical order, and what is more, the removed exceptions should be removed from this config